packages/frontend/src/pages/Dashboard.svelte

```diff
@@ -71,6 +89,26 @@ const openPlaygroundsPage = () => {
           allow for experimenting with available models in a local environment. An intuitive user prompt helps in exploring
           the capabilities and accuracy of various models and aids in finding the best model for the use case at hand.
         </p>
+
+        <h1 class="text-xl first-letter:uppercase underline">Feedback</h1>
+        <p>
+          If you discover an issue or want to request a new feature, you can open an issue
+          <a
+            href="{'javascript:void(0);'}"
+            class="underline"
+            role="button"
+            title="Open the issues page"
+            on:click="{openIssuesPage}">here</a>
+        </p>
+        <p>
+          If you want to get more information about this project or exchange with the team, you can start a discussion
+          <a
```

Comment:
I'm also wondering why you're not using `<Button>` component but then a link with href with JavaScript void

AFAIK we have Link type button